### ISSUE TITLE
Fails to build with Agda 2.6.4

### ISSUE THREAD

**Issue by ncfavier** [ryanorendorff/functional-linear-algebra#47]
```
/build/source/src/FLA/Algebra/LinearMap/Properties.agda:101,7-11
f[u+v]≡f[u]+f[v]ᶜ u v != f[u+v]≡f[u]+f[v]ᵈ u v of type
f (u +ⱽ v) ≡ f u +ⱽ f v
when checking that the expression refl has type
record
{ f = f
; f[u+v]≡f[u]+f[v] = f[u+v]≡f[u]+f[v]ᶜ
; f[c*v]≡c*f[v] = f[c*v]≡c*f[v]ᶜ
}
≡
record
{ f = f
; f[u+v]≡f[u]+f[v] = f[u+v]≡f[u]+f[v]ᵈ
; f[c*v]≡c*f[v] = f[c*v]≡c*f[v]ᵈ
}
```

**Comment by ryanorendorff** [comment]
Thanks! I saw this through the nixpkgs PR for the Haskell updates; I will make the fix soon so it works on that branch.

**Comment by ryanorendorff** [comment]
This PR should fix the problem: #48 . Will push up to the haskell branch once this PR is merged.

**Comment by ryanorendorff** [comment]
Should be fixed now; I'll make an update to the haskell updates branch in nixpkgs.
